tools/Customization/DevHome.Customization/CustomControls/DevDriveCardBody.xaml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<UserControl
+    x:Class="DevHome.Customization.CustomControls.DevDriveCardBody"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+    <!--
+        Grid containing name of dev drive, state of dev drive in text and a list of the dev drive properties.
+    -->
+    <Grid Grid.ColumnSpan="3" Style="{StaticResource CardBodyContainerStyle}">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="*" />
+        </Grid.ColumnDefinitions>
+        <!-- 64x64 image for the dev drive. -->
+        <Viewbox HorizontalAlignment="Left"
+                Grid.Column="0">
+            <Grid>
+                <!-- Image for card body. -->
+                <FontIcon Glyph="&#xEDA2;" FontSize="25" FontFamily="{ThemeResource AmcFluentIcons}"/>
+            </Grid>
+        </Viewbox>
+        <TextBlock Grid.Column="1" Margin="0 5 0 0"
+                Style="{StaticResource CardBodyStackPanelTextBlockStyle}"
+                Text="{x:Bind DevDriveLabel, Mode=OneWay}"
+                IsTextSelectionEnabled="True"/>
+        <Grid Grid.RowSpan="3" Grid.Column="2" HorizontalAlignment="Stretch">
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto"  />
+                <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+                <TextBlock Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" HorizontalAlignment="Right" Margin="0 0 115 5">
+                    <Run> Size - </Run>
+                    <Run Text="{Binding Path=DevDriveSize, Mode=OneWay}" />
+                    <Run Text="{Binding Path=DevDriveUnitOfMeasure, Mode=OneWay}" />
+                </TextBlock>
+            <ProgressBar Grid.Row="1" Width="200" Value="{x:Bind DevDriveFillPercentage, Mode=OneWay}" HorizontalAlignment="Right" />
+            <Grid Grid.Row="2" Grid.ColumnSpan="2">
+                <TextBlock Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" HorizontalAlignment="Right" Margin="0 0 120 0">                        
+                    <Run Text="{Binding Path=DevDriveUsedSize, Mode=OneWay}" />
+                    <Run Text="{Binding Path=DevDriveUnitOfMeasure, Mode=OneWay}" />
+                    <Run> used</Run>
+                </TextBlock>
+                <TextBlock Style="{StaticResource CardBodyStackPanelAltTextBlockStyle}" HorizontalAlignment="Right">                        

```

Comment:
@bbonaby Can you help figure out how to do this without breaking into runs? This is going to be a translation problem if other languages don't put the runs in this order, it should be one string with variables in the .resw. I'm less familiar with what the design should look like so I think you'll be able to advise here better than me.